src/frontend/packages/lib_common/package.json

```diff
@@ -31,6 +33,7 @@
     "@babel/preset-env": "7.20.2",
     "@babel/preset-typescript": "7.18.6",
     "@formatjs/cli": "6.0.1",
+    "@openfun/cunningham-react": "0.1.0",
```

Comment:
as you set in the main packages.json resolution, you should probably use `*` here ?